### PR TITLE
Don't refresh during resync; increase batch size; refresh a bit less during row ops

### PR DESCRIPTION
**Pull request by urmilan**
Implementing first couple of perf improvements we agreed on this morning. The rest are on their way.

### CODE REVIEW

**Comment by znep** [comment]
Do we want to have a default refresh value or do we want to ensure we specify it everywhere?  Or default to true?

**Comment by urmilan** [comment]
@znep: Deleted the default value for refresh in 30d8c45. Also added cleanup of stale data into resync and moved some code into a new class.

File: spandex-secondary/src/main/scala/com.socrata.spandex.secondary/ResyncHandler.scala [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      batch <- iter.grouped(batchSize)
+      row <- batch
+    } {
+      val requests = row.toSeq.collect {
+        case (id, value: SoQLText) =>
+          client.getIndexRequest(RowOpsHandler.fieldValueFromDatum(
+            datasetInfo.internalName, copyInfo.copyNumber, getRowId(row), (id, value)))
+      }
+      // Don't refresh ES during resync
+      client.sendBulkRequest(requests, refresh = false)
```
Comment (velvia) [inline spandex-secondary/src/main/scala/com.socrata.spandex.secondary/ResyncHandler.scala]: If I'm reading this correctly, you are issuing a bulk request, one for every row, is that right?

Comment (chitang) [inline spandex-secondary/src/main/scala/com.socrata.spandex.secondary/ResyncHandler.scala]: Rename the vals a little:
batches <- iter.grouped(batchSize)
batch <- batches

**Comment by urmilan** [comment]
@velvia Good catch, will send a new PR

**Pull request merged**